# HG changeset patch
# User Benoit Jacob <bjacob@mozilla.com>
# Date 1314028164 14400
# Node ID ba3eb3b42f25f08c87088f4f132c2a7f68d5891a
# Parent  72ddbc2378a25651a0b833b078a0e309f62f6b24
Bug 679864 - Upgrade WebGL conformance test suite to r15318

This patch disables the quickCheckAPI.html page, with this comment:

this test causes whichever comes after to intermittently time out.
forcing a GC run doesn't solve this issue. Could be something about using a random amount of memory that
can be too high, causing e.g. swapping.

diff --git a/content/canvas/test/webgl/conformance/more/00_test_list.txt b/content/canvas/test/webgl/conformance/more/00_test_list.txt
--- a/content/canvas/test/webgl/conformance/more/00_test_list.txt
+++ b/content/canvas/test/webgl/conformance/more/00_test_list.txt
@@ -1,12 +1,15 @@
 conformance/constants.html
 conformance/getContext.html
 conformance/methods.html
-conformance/quickCheckAPI.html
+#this test causes whichever comes after to intermittently time out.
+#forcing a GC run doesn't solve this issue. Could be something about using a random amount of memory that
+#can be too high, causing e.g. swapping.
+#conformance/quickCheckAPI.html
 conformance/webGLArrays.html
 functions/bindBuffer.html
 functions/bindBufferBadArgs.html
 functions/bindFramebufferLeaveNonZero.html
 functions/bufferData.html
 functions/bufferDataBadArgs.html
 functions/bufferSubData.html
 functions/bufferSubDataBadArgs.html
